Introduce the localization guide featuring the translation API (#6758)
ckeditor/ckeditor5: docs/framework/guides/architecture/ui-library.md

Views use [templates](#templates) to build the UI. They also provide observable interfaces that other features (e.g. {@link framework/guides/architecture/core-editor-architecture#plugins plugins}, {@link framework/guides/architecture/core-editor-architecture#commands commands}, etc.) can use to change the DOM without any actual interaction with the native API.

<info-box>

> 	All views can be localized using the `locale` instance on which they are created. Check the {@link framework/guides/deep-dive/localization localization guide} to see how to use the `t()` function coming from the `locale` instance.

Reviewer: ```suggestion
	All views can be localized using the `locale` instance with which they were created. Check the {@link framework/guides/deep-dive/localization localization guide} to see how to use the `t()` function available in the `locale` instance.
```
